New cards, bug fixes and official numbers

EerieCode opened the pull request.

EerieCode committed:
Official number for "Predaplant Dragostapelia"

EerieCode committed:
"Predator Craft" changed to "Predator Blast"

Delete the file 100212100.jpg from the pics folder to redownload the
correct image

EerieCode committed:
Bug fixes

Superheavy Samurai Daihachi
Transcendental Polymerization

EerieCode committed:
Added new cards

Astrograph Sorcerer
Chronograph Sorcerer

EerieCode committed:
Added new cards (credits to dest)

True King V.F.D., the Beast

EerieCode committed:
Added new cards (credits to nekrozar)

Brohunder
Number 70: Deadly Sin
Celestial Double Star Shaman
Gozuki
Darklord Ukoback
Frightfur Reborn
Necroid Synchro
Da'at Metatron, the True Dracomechsoldier
Ignis Heat, the True Dracowarrior
Majesty Maiden, the True Dracomage
Dynamite Knuckle, the True Dracofighter
Dragonic Diagram
True Draco Succession
Revival of the True King
Mariamune, the True Dracophoenix

kenpazaraki commented:
Thank you!
I found a bug, though. The True Dracos shouldn't be able to tribute face-down Spell/Trap cards.

EDIT: Also, Dynamite Knuckle and Ignis Heat can't activated the searched card if their effect is activated during the Damage Step.

EerieCode commented:
Does it work with "Mariamune, the True Dracophoenix"? If it does, then you
probably didn't use the modified scripts for the older True Kings:
otherwise, I'll check it as soon as I can.

EerieCode committed:
TCG names and effects for Destiny Soldiers

kenpazaraki commented:
Which effect are you referring to?

EerieCode commented:
Sorry, I've posted in the wrong thread. :P
I'll check those bugs as soon as I can!

EerieCode committed:
Bug fixes

Da'at Metatron, the True Dracomechsoldier
Ignis Heat, the True Dracowarrior
Majesty Maiden, the True Dracomage
Dynamite Knuckle, the True Dracofighter
Revival of the True King

EerieCode commented:
Okay, this should take care of the face-down bug. The part about activation will require more testing.

kenpazaraki commented:
No problem. Also, Frightfur Reborn gives an error:
24: Parameter 2 should be "function"

EerieCode committed:
Fixed "Frightfur Reborn"

EerieCode commented:
All patched up! ;)

EerieCode committed:
Fixed "Elemental HERO Honesty Neos"
